pkgs/development/python-modules/webdriver-manager/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  requests,
+  python-dotenv,
+  tqdm,
+  packaging,
+  pytestCheckHook,
+  selenium,
+}:
+
+buildPythonPackage rec {
+  pname = "webdriver_manager";
+  version = "3.8.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7niNOJuPRSIqimL285tXk2Ch+Hvkba1tqJkYNUrzznM=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    python-dotenv
+    tqdm
+    packaging
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    selenium
+  ];
+
+  # Disabling tests since it requires network
+  doCheck = false;
```

Comment:
All tests? We can disable individual tests with `disabledTests` and `disabledTestPaths`.

Reply:
Just try it out which tests fail. If the majority fails we can keep them disabled.

Reply:
Yeah I will get some information on how to run tests on python modules first and update accordingly.

Reply:
Fixed.